compiler/optimizer/src/main/java/edu/snu/nemo/compiler/optimizer/pass/compiletime/annotating/DecompressionPass.java

```diff
@@ -26,13 +26,14 @@
  * A pass for applying decompression algorithm for data flowing between vertices.
  * It always
  */
+@Annotates(CompressionProperty.class)
 public final class DecompressionPass extends AnnotatingPass {
 
   /**
    * Constructor.
    */
   public DecompressionPass() {
-    super(CompressionProperty.class);
+    super(DecompressionPass.class);
   }
 
```

Comment:
It seems that this pass requires `CompressionProperty`. Why don't we add the property to `Requires` annotation?

Reply:
The author of the pass might have missed that one out then. I'll add it if applicable.